srcs/incs/IRequestHandler.hpp

```diff
@@ -0,0 +1,12 @@
+#ifndef SRCS_INCS_IREQUESTHANDLER_HPP_
+#define SRCS_INCS_IREQUESTHANDLER_HPP_
+#include <string>
+
+class IRequestHandler {
+	public:
+		virtual std::string	GetRawResponse() const = 0;
+		virtual bool		GetKeepAlive() const = 0;
+		virtual				~IRequestHandler() {}
```

Comment:
Implementacion de un metodo puro en una cabecera?
He visto [antes](https://isocpp.org/wiki/faq/pointers-to-members#functionoids) que conviene definir el destructor como puro, y despues implementarlo igualmente, pero no entiendo la logica, y además en este caso, no lo tienes como puro en ningun lado. Está bien? Por qué?

Reply:
Según [C++ Core Guidelines](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c121-if-a-base-class-is-used-as-an-interface-make-it-a-pure-abstract-class), hacer el destructor virtual previene las fugas de memoria
cuando se elimina la clase derivada utilizando el puntero de la interfaz